data/hai/hai reveal 4 middle.txt

```diff
@@ -966,8 +966,35 @@ mission "Hai Reveal [A08] Symbol Identification"
 		log "Giti identified the angry cat symbol. It is from a long-defunct subversive group within the Syndicate called Base Principles. Somehow they obtained alien technology, and used the knowledge gained from it to advance their influence within the organization. A half-century ago, Syndicate Internal Affairs scoured them from the Syndicate's ranks in an extremely thorough and prejudiced inquisition."
 		
 		conversation
-			`After landing, you search up the way to the adjunct research library at the Navy Intelligence Observatory in Moscow. It takes three shuttle journeys and one high-speed rail trip, but you reach the building in under three hours. For someone of your background there are parts of this trip that seem dizzyingly unreal. As well traveled as you now are, actually navigating the urban environments of <origin> away from any particular spaceport is an experience not quite like any other. Once you reach the building in question you take a moment at the reception desk to interpret 'room number: 3017A19' - the 301st floor, section 7A, room 19 - and get directions. It's amazing how large Earth's megabuildings have become. Some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`
+			`You land at Ulaanbaatar's spaceport and receive a message that Batu has to finalize some arrangements and will arrive in the evening. That leaves plenty of time for you to see to your other business about the symbol.`
+			`	At an information desk, you inquire about the route to Moscow, and are given two options. You can take a very expensive private taxi service, which you could bill to Alondo. Or, you can take the long route and see the sights. Either way, you'll have plenty of time before Batu arrives.`
```

Comment:
```suggestion
			`	At an information desk, you inquire about the route to Moscow, and are given two options. You can take a very expensive private taxi service, which you could bill to Alondo. Or, you can take a longer route and see the sights. Either way, you'll have plenty of time before Batu arrives.`
```

Reply:
"The long route" is a figure of speech here. The actual distance is probably about the same.

Reply:
I didn't realise that. I just thought that having to go by 3 shuttles, the player would have to take a more round-about route.

Reply:
The taxi goes out to space and back down, but it spends most of the time in space, so the duration is shorter, but the distance traveled is longer.

We're splitting hairs, though. The intent is to use the figure of speech, not a quantification of the relative distance.